src/ppm.cpp

```diff
@@ -127,58 +138,58 @@ PPM::~PPM()
     RdKafka::wait_destroyed(5000);    // pause to let RdKafka reclaim resources.
 }
 
-void PPM::metadata_print (const std::string &topic, const RdKafka::Metadata *metadata) {
+void PPM::metadata_print (const string &topic, const RdKafka::Metadata *metadata) {
 
-    std::cout << "Metadata for " << (topic.empty() ? "" : "all topics")
+    cout << "Metadata for " << (topic.empty() ? "" : "all topics")
```

Comment:
Same for this full file, how much effort to swap to the logger instead of cout/cerr?

Reply:
It'll be an easy change for any methods that are already a member of the PPM class. Otherwise, we'll just need to make the method a member of that class.

Reply:
Do you think you can fit it in this sprint, or should we add it to the backlog?

Reply:
I can fit it in this sprint.

Reply:
We have swapped to using the logger in this class in all instances except one.